src/modules/design/diagram-detail/diagram-detail.ts

```diff
@@ -286,6 +293,45 @@ export class DiagramDetail {
     }
   }
 
+  public async setOptionsAndStart(): Promise<void> {
+
+    if (this._diagramHasChanged) {
+      this._saveDiagram();
+    }
+
+    const customCorrelationIdIsUndefined: boolean = this.customCorrelationId === undefined;
```

Comment:
Maybe
 
```suggestion
    const noCustomCorrelationId: boolean = this.customCorrelationId === undefined;
```

to keep this in line with `noInitialToken`?